setup.py

```diff
@@ -8,7 +8,7 @@
 
 from setuptools import find_packages, setup
 
-requires = ["click", "cookiecutter", "networkx", "numpy", "pandas", "tornado", "tqdm"]
+requires = ["click", "cookiecutter", "networkx", "numpy", "pandas", "tornado", "tqdm", "seaborn"]
```

Comment:
I will raise this at the dev meeting today, but generally, adding dependencies is a considered a very big deal as we try and keep Mesa as lean as possible to reduce complexity and dependency issues.

Reply:
Yes, I understand. Since matplotlib itself also isn't an dependency, we can also introduce an optional "plotting" dependency which includes matplotlib and seaborn.

Reply:
The dev meeting got pushed to the 15th, however, instead of adding to ore dependencies can you add to the `requirements.txt`? Please see comments in the `.ipynb` file.

Reply:
The `requirements.txt` of Mesa itself or of the [boltzman wealth](https://github.com/projectmesa/mesa-examples/blob/main/examples/boltzmann_wealth_model_network/requirements.txt) model in mesa-examples?

Reply:
Of the Boltzman wealth please this way we keep Mesa lean.

Reply:
Yeah, Matplotlib/Seaborn shouldn't be a required dependency to install Mesa. I think the `ModuleNotFoundError` is a sufficient error message, that we shouldn't have a requirements.txt that is specific to the tutorial.